plugins/account_history/include/golos/plugins/account_history/history_object.hpp

```diff
@@ -53,10 +59,12 @@ namespace golos { namespace plugins { namespace account_history {
         uint32_t block = 0;
         uint32_t sequence = 0;
         operation_id_type op;
```

Comment:
For filtering by an operation name add:
```
uint32_t op_name;
```

Which should be initialized by `op.witch()`